README.md

```diff
@@ -6,7 +6,14 @@ Common JS module for calculating and converting Earth distances using correct
 Installation
 ------------
 
-    npm install geo-distance
+```sh
+npm install geo-distanc
```

Comment:
should be "geo-distance" no?

Reply:
@bhaveshdaswani93 
It's October again, maybe you can finish this PR 😉